index.ts

```diff
@@ -1048,6 +1057,71 @@ commander
   });
 });
 
+/**
+ * Prints out 5 most recent the StackDriver logs
+ * Use --json for output in json format
+ * Use --open to open logs in StackDriver
+ */
+commander
+  .command('logs')
+  .description('Shows the StackDriver Logs')
+  .option('--json', "Show logs in JSON form")
+  .option('--open', 'Open the StackDriver logs in browser')
+  .action((cmd: LoginOptions) => {
+    function printLogs(entries) {
+      for (let i = 0; i < 5; i++) {
+        const severity = `(${entries[i].metadata.severity})`;
+        const timestamp = entries[i].metadata.timestamp;
+        const payloadType = entries[i].metadata.payload;
+        let functionName = entries[i].metadata.resource.labels.function_name;
+        if (functionName) {
+          functionName = functionName.padEnd(15);
+        } else {
+          functionName = ERROR.NO_FUNCTION_NAME;
+        }
+        let payload = '';
+        if (cmd.json) {
+          payload = JSON.stringify(entries[i], null, 4);
+        } else {
+          if (payloadType === 'textPayload') {
+            payload = entries[i].metadata.textPayload;
+          } else if (payloadType === 'jsonPayload') {
+            payload = entries[i].metadata.jsonPayload.fields.message.stringValue;
+          } else if (payloadType === 'protoPayload') {
+            payload = entries[i].metadata.protoPayload;
+          } else {
+            payload = ERROR.PAYLOAD_UNKNOWN;
+          }
+          if (payload && typeof(payload) === "string") {
+            payload = payload.padEnd(20);
+          }
+        }
+        const logString = `${severity.padEnd(10)} - ${functionName} - ${payload} - [${timestamp}]`;
+        console.log(logString);       
+      }
+    }
+  
+    getProjectSettings().then(({ scriptId, rootDir, projectId }: ProjectSettings) => {
+      if (!projectId) {
+        console.error(ERROR.NO_GCLOUD_PROJECT);
+        process.exit(-1);
+      } else {
```

Comment:
You can get rid of this `else` to reduce the nesting. Optionally `return` to explicitly tell the programmer we're stopping.

```ts
if (!projectId) {
  console.error(ERROR.NO_GCLOUD_PROJECT);
  return process.exit(-1);
}
if (cmd.open) {
```